test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -989,3 +988,55 @@ sun/security/x509/X509CRLImpl/OrderAndDup.java https://github.com/eclipse-openj9
 sun/security/x509/X509CRLImpl/Verify.java https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
 sun/security/x509/X509CertImpl/V3Certificate.java https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
 sun/security/x509/X509CertImpl/Verify.java https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
+#
+# Exclude the below tests from sanity.openjdk according to the issues:
+# https://github.ibm.com/runtimes/backlog/issues/1089
+#
+java/security/SignedJar/spi-calendar-provider/TestSPISigned.java https://github.ibm.com/runtimes/backlog/issues/1089 generic-all
```

Comment:
Does this test run in `FIPS140_3_OpenJcePlus` strict profile?
The failure was `Caused by: java.security.NoSuchAlgorithmException: MD5 MessageDigest not available`, I expect same failure if it runs in the strict profile.

Reply:
Done.